### ISSUE TITLE
The `enablePaths` property does not work

### ISSUE THREAD

**Issue by MellKam** [denoland/vscode_deno#827]
**Describe the bug**

I used to use `deno.enablePaths` and it worked fine, but now for some reason it doesn't autocomplete for my specified files.

**To Reproduce**

1. Create following file structure.
- deno/
- -   mod.ts
- main.ts
2. Add settings.json file to .vscode folder with `enablePaths` prop
```json
{
  "deno.enablePaths": ["deno"],
}
```
3. Try copying some code into the deno/mod.ts file and see that there is no autocomplete.
It works fine if you try set to true `deno.enable` property, but I need specifically this forder to be analyzed.

Deno extension logs:
```
Starting Deno language server...
  version: 1.31.3 (release, x86_64-unknown-linux-gnu)
  executable: /home/melkam/.cargo/bin/deno
Connected to "Visual Studio Code" 1.76.2
Enabling import suggestions for: https://deno.land
Enabling import suggestions for: https://x.nest.land
Download https://x.nest.land/.well-known/deno-import-intellisense.json
Download https://intellisense.nest.land/deno-import-intellisense.json
Enabling import suggestions for: https://crux.land
Server ready.
client asked to cancel request 24, but no such pending request exists, ignoring
client asked to cancel request 45, but no such pending request exists, ignoring
client asked to cancel request 71, but no such pending request exists, ignoring
```

**Expected behavior**

Analyze and autocomplete my code :)

**Screenshots**

![image](https://user-images.githubusercontent.com/51422045/226101571-44e93166-896d-4ec5-a066-2139208c2456.png)

**Versions**

Extension: 3.17.0
VsCode: 1.76.2
Deno: 1.31.3

**Comment by dsherret** [comment]
Does it work in 1.31.2? I think this is a new bug in 1.31.3. It should be fixed in 1.32.1 via https://github.com/denoland/deno/pull/18373

**Comment by MellKam** [comment]
> Does it work in 1.31.2? I think this is a new bug in 1.31.3. It should be fixed in 1.32.1 via https://github.com/denoland/deno/pull/18373

Yes, it worked with deno 1.13.2. Thanks for fixing this 🙏

**Comment by dsherret** [comment]
Cool. I think it's fixed now in 1.32.1 (released as of yesterday). Let me know if not.
